tests/phpthemis/composer-setup.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash -e
 
 php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+php -r "echo 'Installed hash = '; echo hash_file('SHA384', 'composer-setup.php');"
 
-php -r "if (hash_file('SHA384', 'composer-setup.php') === '93b54496392c062774670ac18b134c3b3a95e5a5e5c8f1a9f115f203b75bf9a129d5daa8ba6a13e2cc8a1da0806388a8') 
+php -r "if (hash_file('SHA384', 'composer-setup.php') === '48e3236262b34d30969dca3c37281b3b4bbe3221bda826ac6a9a62d6444cdb0dcd0615698a5cbe587c3f0fe57a54d8f5') 
   { echo 'Valid signature, continue installation'; } 
 else 
-  { echo 'ERROR: Invalid installer signature, probably installer was updated'; unlink('composer-setup.php'); exit(1); } 
+  { echo 'ERROR: Invalid installer signature, probably installer was updated, fix in /tests/phpthemis/composer-setup.sh'; unlink('composer-setup.php'); exit(1); } 
```

Comment:
Pointing to the file with most likely fix is really helpful. Thank you.

Reply:
Previous time when installed was updated past me spent ~hour to figure out.
Current me spent ~10 min.
Future me should spend ~1 min.